client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/client/issue/ShowIssueParams.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.client.issue;
+
+import java.util.List;
+import org.sonarsource.sonarlint.core.clientapi.common.FlowDto;
+import org.sonarsource.sonarlint.core.clientapi.common.TextRangeDto;
+
+public class ShowIssueParams {
+  private final String fileUri;
+  private final String message;
+  private final String connectionId;
+  private final String ruleKey;
+  private final String issueKey;
+  private final String creationDate;
+  private final String codeSnippet;
+  private final List<FlowDto> flows;
+  private final TextRangeDto textRange;
```

Comment:
You probably don't need it in VSCode, but here we need to have the configuration scope ID. 
I'm not sure the connectionId is required because if you have the configScopeId you can retrieve the connection on the client side.

Reply:
Addressed in [383cbb5](https://github.com/SonarSource/sonarlint-core/pull/722/commits/383cbb5505665f2da97a4c2f86511f34a228339c)